actionsystemtest/lib/action_system_test/server.rb

```diff
@@ -0,0 +1,34 @@
+require "rack/handler/puma"
```

Comment:
The decision was made to move this all back into Action Pack and not add hard dependencies so no it shouldn't be a runtime dependency.